.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,43 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the
+// README at: https://github.com/devcontainers/templates/tree/main/src/docker-existing-docker-compose
+{
+	"name": "Actual development",
+
+	// Update the 'dockerComposeFile' list if you have more compose files or use different names.
+	// The .devcontainer/docker-compose.yml file contains any overrides you need/want to make.
```

Comment:
Can you remove the comments like these unless they’re explaining something that a person reading this would need to know?

Reply:
I wasn't sure about leaving them in or not, as the maintainers might not have been aware of the options.
Will remove (besides the image one)

Reply:
They were definitely useful to look at on the first review pass, thanks for keeping them in! But yes would prefer to keep the final files clean